Include/object.h

```diff
@@ -491,6 +491,24 @@ static inline void Py_INCREF(PyObject *op)
 }
 #define Py_INCREF(op) Py_INCREF(_PyObject_CAST(op))
 
+/// Increment reference counter by specified number of items
+static inline void Py_INCREF_n(PyObject* op, Py_ssize_t n)
```

Comment:
This should not go into the public API. Then you shouldn't need the first `#ifdef` either.

Reply:
Done